### PR TITLE
Fix AutoRound scale dtype UT

### PR DESCRIPTION
**Pull request by Kaihui-intel**
## Type of Change

bug fix

## Description
Fix AutoRound accuracy gap between quantized model and export model on spr/gpu

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by yiliu30 — APPROVED** [review]

File: requirements_pt.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-auto-round
+auto-round @ git+https://github.com/intel/auto-round.git@eaf3710ca1df0a645acbe97e965f8672ff8ba341
```
Comment (yiliu30) [inline requirements_pt.txt:1]: This requirement file is used by INC3 torch. Please update it in a separate PR if needed.

Comment (chensuyue) [reply requirements_pt.txt:1]: v0.11 has been released, we can remove the version limit.

Comment (Kaihui-intel) [reply requirements_pt.txt]: requirements_pt.txt not updated,  so no other PR is needed.

Comment (Kaihui-intel) [reply requirements_pt.txt]: removed the version limit https://github.com/intel/neural-compressor/pull/1688/commits/feae5ed7a4d3d49a82ad7c1a6e01cacae07e206c

**Comment by chensuyue** [comment]
> # ⛈️ Required checks status: Has failure 🔴
> > **Warning**
> > If you do not have the access to re-run the Probot, please contact [XuehaoSun](https://github.com/XuehaoSun) for help. If you push a new commit, all of the workflow will be re-triggered.
> 
> ## Groups summary
> 🟢 Code Scan Tests workflow
> 🟢 Model Tests workflow
> 🟢 Unit Tests basic workflow
> 🟢 Unit Tests basic no coverage workflow
> 🔴 Unit Tests ITREX workflow
> Check ID	Status	Error details	
> [UT-ITREX](https://dev.azure.com/lpot-inc/b7121868-d73a-4794-90c1-23135f974d09/_build/results?buildId=28051)	failure	[download](https://artprodcus3.artifacts.visualstudio.com/Acd5c2212-3bfc-4706-9afe-b292ced6ae69/b7121868-d73a-4794-90c1-23135f974d09/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2xwb3QtaW5jL3Byb2plY3RJZC9iNzEyMTg2OC1kNzNhLTQ3OTQtOTBjMS0yMzEzNWY5NzRkMDkvYnVpbGRJZC8yODA1MS9hcnRpZmFjdE5hbWUvMV91dF9pdHJleF9yZXBvcnQ1/content?format=file&subPath=%2Fut_itrex.log)	❌
> These checks are required after the changes to `neural_compressor/adaptor/pytorch.py`.
> 
> Thank you for your contribution! 💜
> 
> > **Note**
> > This comment is automatically generated and will be updates every 180 seconds within the next 6 hours. If you have any other questions, contact [chensuyue](https://github.com/chensuyue) or [XuehaoSun](https://github.com/XuehaoSun) for help.

ITREX CI issue may caused by onnx version upgrade, https://pypi.org/project/onnx/#history, @mengniwang95 will help to check.

**Pull request merged**